server/constants/oauth_config.go

```diff
@@ -13,5 +13,6 @@ const (
 	// Paths
 	PathAuth = "/oauth2/authorize"
 	// #nosec G101 -- This is a false positive
-	PathToken = "/oauth2/token"
+	PathToken       = "/oauth2/token"
+	PathUserProfile = "/_apis/profile/profiles/%s"
```

Comment:
I think this should be in routes.go as this is an AZD API path

Reply:
this is an AZD path but it is used for oAuth purpose only also it uses the base URL of oAuth which is different from the rest